governance.md

```diff
@@ -47,6 +47,12 @@ rule sets. Contributors with a record of consistent, responsible past
 contributions who are planning major contributions in the future could be
 considered to become qualified Maintainers.
 
+To be added as a maintainer to a repository in the bazelbuild organization,
```

Comment:
This may work better as a numbered list.

Reply:
Addressed.